Utilities/AzureOnboardingCLI/azure-automatic-role-assignment.py

```diff
@@ -410,11 +445,16 @@ def login_to_azure():
     Returns:
     None
     """
-    print("Please login to azure (Web page should be opened)")
+    log("Please login to azure (Web page should be opened)")
     azure_login_cmd = "az login"
     run_command(azure_login_cmd)
 
 
+def ensure_automatic_extension_install():
+    run_command('az config set',
```

Comment:
Comment - what is this for?

Reply:
Added comment.

Reply:
Minor, but based on the comment, I would even go `ensure_azure_cli_automatic_extension_install_enabled`.